app/controllers/action_page_controller.rb

```diff
@@ -109,6 +109,7 @@ def set_action_display_variables
     @tweet = @actionPage.tweet
     @email_campaign = @actionPage.email_campaign
     @congress_message_campaign = @actionPage.congress_message_campaign
+    @partner = @actionPage.random_partner
```

Comment:
This is choice is going to cached for some time behind Varnish. We should follow up and move the partner selection to javascript.